mpas_analysis/ocean/nino34_index.py

```diff
@@ -66,6 +67,17 @@ def nino34_index(config, streamMap=None, variableMap=None):  # {{{
     # reading only those that are between the start and end dates
     startDate = config.get('index', 'startDate')
     endDate = config.get('index', 'endDate')
+    dataPath = config.get('indexNino34', 'observationData')
+
+    # specify obsTitle based on data path
+    # These are the only data sets supported
+    if 'had' in dataPath.lower():
+        obsTitle = 'HADSST'
+    elif 'sstv3b' in dataPath.lower():
+        obsTitle = 'ERS SSTv3b'
+    else:
+        obsTitle = 'ERS SSTv4'
+
```

Comment:
These obsTitle values should actually be what the user selects (e.g. with my suggested `plotObservations` option above).  You would then loop over the observations to plot, rather than selecting one.